generated/nidcpower/_library.py

```diff
@@ -311,14 +310,6 @@ def niDCPower_MeasureMultiple(self, vi, channel_name, voltage_measurements, curr
                 self.niDCPower_MeasureMultiple_cfunc.restype = ViStatus  # noqa: F405
         return self.niDCPower_MeasureMultiple_cfunc(vi, channel_name, voltage_measurements, current_measurements)
 
-    def niDCPower_ParseChannelCount(self, vi, channels_string, number_of_channels):  # noqa: N802
```

Comment:
I thought we were generating this and using it as a private function, it's needed for figuring out memory allocation in niDCPower_MeasureMultiple()

Reply:
Fixed filtering in metadata generation